### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by andllrew**
**Ticket:** [WEB- Create the landing page for the store site](https://cs375-pos-group-project.atlassian.net/jira/core/projects/ATS/board?selectedIssue=ATS-30)

**PR Desc:**
- Created a landing page for our website
- Implemented Bootstrap to stylize
- Added an about and contact section
- Added a button that will allow the user to enter the website and see the items being sold (not yet implemented/another ticket)

**Testing:**
- Run `npm i`
- Run `npm start`
- Visit http://localhost:3000/store/
- Make sure the navbar works
- Clicking the brand should also bring you back to the top

### CODE REVIEW

File: app/frontend/public/store/index.html [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            </div>
+        </section>
+
+        <script
+            src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
+            integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+            crossorigin="anonymous"
+        ></script>
+    </body>
+</html>
```
Comment (wetefok) [inline app/frontend/public/store/index.html:82]: ay fix this

File: app/frontend/public/store/style.css [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    object-position: center;
+}
+
+#about, #contact {
+    height: 60vh;
+}
+
+#about h1, #contact h1 {
+    margin-bottom: 4rem;
+}
```
Comment (wetefok) [inline app/frontend/public/store/style.css:71]: bruuuuuuuh
\n my dude

**Review by wetefok — CHANGES_REQUESTED** [review]
cmon bruh

**Review by wetefok — APPROVED** [review]
☝︎□︎□︎♎︎ ⬧︎♒︎♓︎⧫︎ ♋︎■︎♎︎❒︎♏︎⬥︎📪︎ ⍓︎□︎◆︎ ❒︎♏︎♋︎●︎●︎⍓︎ ♍︎□︎□︎🙵♏︎♎︎ ◆︎◻︎ ⧫︎♒︎♏︎ ❍︎□︎■︎🙵♏︎⍓︎ ⬥︎♓︎⧫︎♒︎ ⧫︎♒︎♓︎⬧︎ □︎■︎♏︎📬︎ ❄︎♒︎♓︎⬧︎ ⧫︎❒︎◆︎●︎⍓︎ ♓︎⬧︎ ♋︎■︎ ♋︎◻︎♏︎⬧︎ ⧫︎□︎♑︎♏︎⧫︎♒︎♏︎❒︎ ⬧︎⧫︎❒︎□︎■︎♑︎ ♒︎□︎❍︎♏︎ ◻︎♋︎♑︎♏︎📬︎ ✌︎●︎●︎ □︎♐︎ ⧫︎♒︎♏︎ ♓︎❍︎♋︎♑︎♏︎⬧︎ ♋︎❒︎♏︎ ⧫︎♋︎⬧︎⧫︎♏︎♐︎◆︎●︎●︎⍓︎ ⬧︎♏︎●︎♏︎♍︎⧫︎♏︎♎︎ ♋︎■︎♎︎ ♌︎❒︎♓︎■︎♑︎ ♋︎ ♍︎♏︎❒︎⧫︎♋︎♓︎■︎ ♑︎❒︎♋︎❖︎♓︎⧫︎♋︎⬧︎ ⧫︎□︎ ⧫︎♒︎♏︎ ⬥︎♏︎♌︎⬧︎♓︎⧫︎♏︎📬︎ 🕈︎♏︎●︎●︎ ♎︎□︎■︎♏︎ ♋︎◻︎♏︎📬︎ 🦍🦍🦍🦍🦍🦍🦍🦍🦍🦍🦍🦍🦍🦍🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊🙊

**Review by nhokzminh698 — APPROVED** [review]

**Pull request merged**